docs/developer-docs/index.md

```diff
@@ -25,4 +27,12 @@ Regardless of the development workflow you choose, keep in mind that when you de
 
 ## Gas cost / Pricing
 
-The Internet Computer is one of the cheapest blockchains that exist today. Storing one GB of data costs around 5$ per year, and for just one cent you can perform around 10.000 transactions. To see the full gas cost table, go [here](./gas-cost.md).
\ No newline at end of file
+The Internet Computer is one of the cheapest blockchains that exist today. 
+
+On the Internet Computer, a "cycle" is the unit of measurement for resources consumed in the form of processing, memory, storage, and network bandwidth. Every canister has a cycles account to which resources consumed by the canister are charged. The Internet Computer’s utility token (ICP) can be converted to cycles and transferred to a canister. Cycles can also be transferred between canisters by attaching them to an inter-canister message.
+
+ICP can always be converted to cycles using the current price of ICP measured in [SDR](https://en.wikipedia.org/wiki/Special_drawing_rights) using the convention that one trillion cycles correspond to one SDR.
+
+**Storing one GB of data costs around 5$ per year, and for just one cent you can perform around 10.000 transactions. **
```

Comment:
should we use the `,` notation here or stick with the european convention?

Reply:
typically, these conversions would be automated by a localization library. we don't have one installed at the moment, but we certainly may in the future.

Reply:
changed it to `10_000` which is more universal